apps/builder/app/builder/features/style-panel/style-sections.tsx

```diff
@@ -42,7 +43,7 @@ export const categories = [
   "boxShadows",
   "outline",
   "effects",
-  "transitions",
+  // "transitions",
```

Comment:
should probably move it to the right place? or is it for convenience while in dev?

Reply:
Yup, need to be changed. Was moved just for dev convenience 👍